asciidoc/workloads/workloads.adoc

```diff
@@ -0,0 +1,31 @@
+= Introduction to workloads
+
+What you can do with Ensono Stacks is described in the link:../[About Ensono Stacks] section.
```

Comment:
Do we need to correct this link? Its not really linking to anything. Even the current docs links to just the homepage. I would have thought that it should be linking to here really: https://stacks.ensono.com/docs, which would be this file: docs/index.md